metadata-ingestion/tests/integration/kafka-connect/test_kafka_connect.py

```diff
@@ -62,12 +71,33 @@ def test_kafka_connect_ingest(docker_compose_runner, pytestconfig, tmp_path, moc
             ).status_code
             == 200,
         )
+        yield docker_services
+
+
+@pytest.fixture(scope="module")
+def test_resources_dir(pytestconfig):
+    return pytestconfig.rootpath / "tests/integration/kafka-connect"
 
-        # Creating MySQL source with no transformations , only topic prefix
-        r = requests.post(
-            "http://localhost:58083/connectors",
-            headers={"Content-Type": "application/json"},
-            data="""{
+
+@pytest.fixture(scope="module")
+def loaded_kafka_connect(kafka_connect_runner):
+    # Set up the container.
+    time.sleep(10)
```

Comment:
will this make the kafka-connect tests more stable? we've been seeing a lot of flakes with those tests lately

Reply:
well, the init code is same as before. One improvement is that -> mongo kakfa_connect integration test will become more stable, as it would have same fix (calling docker compose runner twice) as that was present for the other test.

Reply:
This additional delay (borrowed from mongo db test setup) might help for stability of all the tests.
If you can share the link pointing to kafka connect test flakes, I can look into the them to confirm, if this PR will fix them.

Reply:
Such CI failures - in mongo kafka connect source - https://github.com/datahub-project/datahub/pull/7568/checks?check_run_id=11974920401 should be fixed by this PR.

Reply:
nice